src/integration/common/header.js

```diff
@@ -8,12 +8,22 @@ export default () => {
       expect(logo.parentNode.textContent).toMatchSnapshot();
     });
 
+    it("I can see the offscreen text with product's language code set to English", () => {
+      const langCode = document.querySelector(
+        'header div span span[lang="en-GB"]',
```

Comment:
is this `en-GB` for all services?

Reply:
yes except news, scotland and archive.Our tests are not run against these services.